webpack.umd.config.js

```diff
@@ -24,6 +24,7 @@ const webpackUMDConfig = {
     'react-dom': 'ReactDOM',
   },
   plugins: [
+    new webpack.DefinePlugin(config.compiler_globals),
```

Comment:
This might also be a bad merge.  I don't think we make use any globals in the /src.  Those would be found as double underscored variables (i.e. `__DEV__`).